### PR TITLE
update functionality.

### PR DESCRIPTION
**Pull request by sjpadgett**
Add create and search encounters.
Some mod's to provider client.
These resources will still need tuning once we figure out how the heck we'll use them:) But 'ya got to admit we're much closer to usable FHIR.

### CODE REVIEW

File: phpfhir/providerPublishUI.php [hunk 1]
```diff
@@ -89,16 +99,31 @@ function doPublish(e, req) {
     </div><!-- /.container-fluid -->
 </nav>
 <div class="container-fluid main-container">
+    <div class="row">
+        <form>
+            <div class="col-md-2 content form-group">
+                <div class="input-group input-group-sm">
+                    <label for="enc">Resource (select one):</label>
```
Comment (bradymiller) [inline phpfhir/providerPublishUI.php]: translate

File: phpfhir/providerPublishUI.php [hunk 2]
```diff
@@ -89,16 +99,31 @@ function doPublish(e, req) {
...
+                    <select class="form-control" id="resource">
+                        <option value="Patient" selected>Patient (This)</option>
```
Comment (bradymiller) [inline phpfhir/providerPublishUI.php]: translate

File: phpfhir/providerPublishUI.php [hunk 3]
```diff
@@ -89,16 +99,31 @@ function doPublish(e, req) {
...
+                        <option value="Encounter">Encounters (All)</option>
```
Comment (bradymiller) [inline phpfhir/providerPublishUI.php]: translate

**Comment by sjpadgett** [comment]
@bradymiller I'll go ahead and translate these but the dashboard client will be evolving and is more of a way to test. In the end the search and reads will need to be parsed and presented in a better form for most user's whom have no idea what json is. I'm mainly interested in what profiles/resources we are going to need and getting them finished. I'm thinking the provider client will turn into a chart publish mainly.

**Comment by bradymiller** [comment]
@sjpadgett , Can't help myself on the translations; it's an illness :)

**Pull request merged**